common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1440,6 +1450,55 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(8)
 
+  val PARTITION_SPLIT_THRESHOLD: ConfigEntry[Long] =
+    buildConf("celeborn.shuffle.partitionSplit.threshold")
+      .withAlternative("rss.partition.split.threshold")
+      .categories("client")
+      .doc("Shuffle file size threshold, if file size exceeds this, trigger split.")
+      .version("0.2.0")
+      .bytesConf(ByteUnit.BYTE)
+      .createWithDefaultString("256m")
+
+  val PARTITION_SPLIT_MODE: ConfigEntry[String] =
+    buildConf("celeborn.shuffle.partitionSplit.mode")
+      .withAlternative("rss.partition.split.mode")
+      .categories("client")
+      .doc("soft, the shuffle file size might be larger than split threshold ; hard, the shuffle file size will be limited to split threshold.")
+      .version("0.2.0")
+      .stringConf
+      .checkValue(
+        value => Seq("soft", "hard").contains(value.toLowerCase(Locale.ROOT)),
+        s"Invalid split mode, Celeborn only support partition type of (soft, hard)")
+      .createWithDefault("reduce")
+
+  val BATCH_HANDLE_CHANGE_PARTITION_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.shuffle.batchHandleChangePartition.enabled")
+      .withAlternative("rss.partition.split.threshold")
+      .categories("client")
+      .doc("When true, Celeborn support handle change partition request in batch.")
```

Comment:
could you please explain more about the difference between batch enabled/disabled?